src/components/Beneficiaries/index.tsx

```diff
@@ -410,11 +406,10 @@ class BeneficiariesComponent extends React.Component<Props, State> {
     };
 
     private handleSetCurrentAddressOnUpdate = (beneficiaries: Beneficiary[], currency: string) => {
-        const filteredByCurrency = this.handleFilterByCurrency(beneficiaries, currency);
```

Comment:
If you removed usage of handleFilterByCurrency function, you can remove currency from arguments of handleSetCurrentAddressOnUpdate as soon as it's not used in the function